src/cephadm/cephadm

```diff
@@ -2358,6 +2359,7 @@ class CephContainer:
             'run',
             '--rm',
             '--net=host',
+            '--log-driver=journald',
```

Comment:
This really only affects containers that are run in a detached `-d` state (e6792f306ab4d07251588fdca6ed3876ae3a092a)